### DISCUSSION TITLE
Delay for resizing the window

### DISCUSSION THREAD

**Discussion by xkmanlordx** [flxzt/rnote#1140] [Ideas]
First of big fan.

One minor annoyance though: whilst using my tablet to take notes, when trying to press the save button, or open the sidebar, due to slight imprecision, the window resizes instead of clicking the symbol i was aiming for. 
Maybe introducing a delay for the resize feature could help, although I can see how that could also be annoying, perhaps the hit-boxes could be made slightly larger, but I have no programming knowledge, so I'm just spitballing.

Either way, great app, many thanks for putting it out there.

**Comment by Doublonmousse** [comment]
Moved to #1142

Basically, events that are not handled by the buttons themselves are transferred to the titlebar, so some events can resize the window when clicking on buttons. I'm trying a PR to fix this (do not pass events to the titlebar)
